espnet2/asr_transducer/joint_network.py

```diff
@@ -41,11 +44,18 @@ def __init__(
         self.joint_activation = get_activation(
             joint_activation_type, **activation_parameters
         )
+        # biasing
+        self.joint_space_size = joint_space_size
+        self.deepbiasing = deepbiasing
+        self.biasing = biasing
+        if biasing and deepbiasing:
+            self.lin_biasing = torch.nn.Linear(biasingsize, joint_space_size)
 
     def forward(
         self,
         enc_out: torch.Tensor,
         dec_out: torch.Tensor,
+        hptr: torch.Tensor = None,
```

Comment:
a bit concern is that `hptr` is before `no_projection`. Will this order change affect other transducer models?

Reply:
Thanks for the comments! I have now addressed the problem in asr.sh, and for this, I moved `hptr` after `no_projection` and changed various places where this forward is invoked.